net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/messages.properties

```diff
@@ -20,7 +20,7 @@ CheckConfigurationConfigureDialog_lblConfiguredModules = Configured modules for
 
 CheckConfigurationConfigureDialog_lblDescription = Description:
 
-CheckConfigurationConfigureDialog_lblKnownModules = Known modules
+CheckConfigurationConfigureDialog_lblKnownModules = Available modules
```

Comment:
is it problem to change key ?
to be `CheckConfigurationConfigureDialog_lblAvailableModules`

Reply:
No problem. Be aware there are no more blanks in the properties file, because that was changed automatically on refactoring the name of the Java member. Stuff like that is why I argued everything needs to be reformatted once.

Reply:
please send PR with reformatting all.